resources/shuttle-secrets.mdx

```diff
@@ -11,4 +11,25 @@ with the secrets you'd like to store. Make sure to add `Secrets.toml` to a `.git
 Next, pass `#[shuttle_secrets::Secrets] secret_store: SecretStore` as an argument to your `shuttle_service::main` function.
 `SecretStore::get` can now be called to retrieve your API keys and other secrets at runtime.
 
+##Example
+
+An example of the rocket function that uses the shuttle main service is as follows:
```

Comment:
There needs to be a space between the '#' and "Example" for it to be turned into a header. If you're not sure how to build the docs locally for development, I left some instructions in the [readme](https://github.com/shuttle-hq/shuttle-docs/blob/main/README.md)
```suggestion
## Example

This snippet shows a shuttle rocket main function that uses the `shuttle_secrets::Secrets` attribute to gain access to a `SecretStore`.
```